navigation.json

```diff
@@ -0,0 +1,39 @@
+{
+"header" : {
+    "goToHomeScreenLogo" : ".vs-c-site-logo",
+    "showSearchButton" : ".vs-c-project-search .vs-c-btn",
+    "hideSearchButton" : ".vs-is-active .vs-c-btn",
+    "searchInput" : ".vs-c-project-search__input input[type='text']",
+
```

Comment:
Blank space

Reply:
@bdimitrovski Does the "blank space" refer to empty space between classes? If so, these are mostly classes from different elements (parent el class + child el class). ChroPath doesn’t recognize elements if there's no space between classes.

Reply:
Yes, blank space between `searchInput` and the next element. Nothing major, but still would be better not to have blank lines unnecessarily if possible.